src/omnicore/rpc.cpp

```diff
@@ -78,7 +78,7 @@ void MetaDexObjectToJSON(const CMPMetaDEx& obj, Object& metadex_obj)
     // add data to JSON object
     metadex_obj.push_back(Pair("address", obj.getAddr()));
     metadex_obj.push_back(Pair("txid", obj.getHash().GetHex()));
-    if (obj.getAction() == 4) metadex_obj.push_back(Pair("ecosystem", isTestEcosystemProperty(obj.getProperty()) ? "Test" : "Main"));
+    metadex_obj.push_back(Pair("ecosystem", isTestEcosystemProperty(obj.getProperty()) ? "test" : "main"));
     metadex_obj.push_back(Pair("propertyidforsale", (uint64_t) obj.getProperty()));
```

Comment:
Note, we don't provide an 'ecosystem' attribute in RPC output for any other call - the only reason it was included here was for cancel-by-ecosystem transactions to show what ecosystem the transaction applied to.

I don't really see the need to supply the ecosystem attribute except for that explicit case as it's an API and the ecosystem is easily devolved from already provided property ID data.  But mostly I don't like doing it just for one type of transaction only (MetaDEx trades) - we should either supply property ID only (preferred) or both property ID and ecosystem for all transactions.